app/assets/stylesheets/application.css.scss

```diff
@@ -104,3 +104,8 @@ i.fa.medium {
   transform: translateX(-50%);
   z-index: 997;
 }
+
+// darkens text for input placeholders
+::-webkit-input-placeholder {
+    color: #565656 !important;
```

Comment:
[Hound fix] (https://github.com/OperationCode/operationcode/pull/640/commits/2cfb83149885368f19884519d240443124db7e92)
[In PR](https://github.com/OperationCode/operationcode/pull/639)